.travis.yml

```
@@ -15,7 +15,10 @@ env:
 before_install:
   - git fetch --tags
 
-script: sbt ++$TRAVIS_SCALA_VERSION mimaReportBinaryIssues test scalafmtCheckAll scalafmtSbtCheck
+script: sbt ++$TRAVIS_SCALA_VERSION mimaReportBinaryIssues coverage test scalafmtCheckAll scalafmtSbtCheck
+
+after_success:
+  - sbt coverageAggregate coveralls
```

missing the scala version. as it is, it always tries to aggregate for the default version (2.13) rather than the one we just tested

```suggestion
  - sbt ++$TRAVIS_SCALA_VERSION coverageAggregate coveralls
```

---

Well spotted - thanks! Seems to work now: https://coveralls.io/builds/33253505